### PR TITLE
Alexander Said w4 JavaScript

### PR DESCRIPTION
**Pull request by AlexanderSaid**

### CODE REVIEW

File: 1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js [hunk 1]
```diff
@@ -11,13 +11,9 @@ Let's rewrite it (or _refactor_ it, as experienced developers would call it):
...
-  // TODO rewrite the function body using `map` and `filter`.
-  const newNumbers = [];
-  for (let i = 0; i < numbers.length; i++) {
-    if (numbers[i] % 2 === 0) {
-      newNumbers.push(numbers[i] * 2);
-    }
-  }
+  const newNumbers = numbers
+    .filter((num) => num % 2 === 0)
+    .map((num) => (num *= 2));
```
Comment (killoncio) [inline 1-JavaScript/Week4/homework/ex1-doubleEvenNumbers.test.js:16]: Well done!

I think here you don't need the `=` in `num *= 2`. You just need to return `num * 2`

File: 1-JavaScript/Week4/homework/ex3-lemonAllergy.test.js [hunk 2]
```diff
@@ -26,26 +26,25 @@ const fruitBasket = [
 ];
 
 // ! Function under test
-function sanitizeFruitBasket(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function sanitizeFruitBasket(arr, item) {
```
Comment (killoncio) [inline 1-JavaScript/Week4/homework/ex3-lemonAllergy.test.js:29]: Well done!

File: 1-JavaScript/Week4/homework/ex5-wallet/index.js [hunk 3]
```diff
@@ -26,9 +26,9 @@ function createWallet(name, cash = 0) {
 
   function transferInto(wallet, amount) {
     console.log(
-      `Transferring ${eurosFormatter.format(amount)} from ${name} to ${
-        wallet.name
-      }`
+      `Transferring ${eurosFormatter.format(
```
Comment (killoncio) [inline 1-JavaScript/Week4/homework/ex5-wallet/index.js:29]: Well done again!!

**Comment by AlexanderSaid** [comment]
Hey @killoncio,
Thanks for your review and comments, and sorry for not react earlier.
I've applied your notes in my code.

**Comment by killoncio** [comment]
Perfect, thanks!!

**Pull request closed without merging**